docs/source/chatops/aliases.rst

```diff
@@ -166,7 +166,7 @@ A single alias file allow multiple formats to be specified for a single alias e.
     ---
     name: "st2_sensors_list"
     action_ref: "st2.sensors.list"
-    description: "List available StackStorm sensors."
```

Comment:
wrong: this won't work as |st2| won't expand in code blocks. 
Just keep it as StackStorm.